content/correspondence/getting-started/developer-guides/notifications/_index.en.md

```diff
@@ -0,0 +1,89 @@
+A list of keywords will soon be implemented in Altinn Notifications, but it is not available yet. 
+This feature will allow you to incorporate keywords into the text, such as \$sendersName\$ to display the name of the organization that sent the correspondence, 
+or \$recipientName\$ to use the name of the individual or organization receiving the correspondence. Further details will be provided when it is ready.
+
+## Notification Templates 
+Two types of notification templates are offered when using notifications through the Messaging API. 
+{{% notice warning  %}}
+NOTE: These templates is not the final version. And will change, especially when keywords is ready.
+{{% /notice %}}
+
+**CustomText:**
+
+- The entire message is written by the sender and sent in full to the recipient.
+
+**GenericAltinnMessage:**
+
+A generic Altinn text with the option to supplement with additional text. Currently supported languages are Norwegian, Nynorsk, and English. The language is chosen based on the language defined in the message.
+
+**Title:** You have received a message in Altinn {textToken}<br>
+**Content:** Hello \$recipientName\$, you have received a new message in Altinn from \$sendersName\$. {textToken} Log in to Altinn inbox to see this message.
+
+**Reminder Title:** Reminder - you have received a message in Altinn {textToken}<br>
+**Reminder Content:** Hello \$recipientName\$, this is a reminder that you have received a new message in Altinn from \$sendersName\$. {textToken} Log in to Altinn inbox to see this message.
+
+In the text, textToken will be replaced with the value given in, for example, "EmailSubject" for the title. SMS uses only the content, not the title.
+
+Note! Links should NEVER be used in notifications.
+
+## Notification Channels 
+Supported notification channels:
+
+- **Email:** Sends an email to the recipient.
+- **SMS:** Sends an SMS to the recipient. Supports both national and international phone numbers.
+- **EmailPreferred:** Uses email as the main communication channel, and SMS as a fallback if email is not available.
+- **SmsPreferred:** Uses SMS as the main communication channel, and email as a fallback if SMS is not available.
+The first notification and the reminder notification can use different notification channels. For example, the first notification is sent by email, while the reminder notification seven days later is sent by SMS.
+## Cancellation of Notification
+If a message is deleted before the publication date, the notification order will also be deleted. In situations where an error occurs during the publication of a message, the notification will also be deleted.
+
+## Errors in Notification Ordering
+If no contact information is available for a recipient, the message will still be sent as planned. Information about the notification can be viewed by retrieving the details about the specific correspondence. Improvements are planned to provide feedback on this during the creation of a message.
```

Comment:
Have we considered failing the correspondence initialization when notification order is not possible? Do we know how big this problem is "IRL"? I.e, is it such that anyone who has ever used Altinn has an email, or phone no, registered?

Reply:
I have discussed this with Ragnar, and we decided to replicate the Altinn 2 functionality. 

I will see if we can look at some data about how big the issue actially is